application/src/test/java/dev/ikm/komet/sampler/controllers/SamplerReadOnlyStringController.java

```diff
@@ -34,17 +33,17 @@ public void initialize()
                 "KlIntegerField, KlFloatField, KlUuidField, KlInstantField");
 
         titleTextField.setText(readOnlyStringControl.getTitle());
-        textTextField.setText(readOnlyStringControl.getText());
+        textTextField.setText(readOnlyStringControl.getValue());
         editModeCheckBox.setSelected(readOnlyStringControl.isEditMode());
 
         // Data Type Combobox
-        for (StringDataType stringDataType : StringDataType.values()) {
-            dataTypeComboBox.getItems().add(stringDataType);
-        }
-        dataTypeComboBox.setValue(readOnlyStringControl.getDataType());
-        dataTypeComboBox.valueProperty().addListener(observable -> {
-            readOnlyStringControl.setDataType(dataTypeComboBox.getValue());
-        });
+//        for (DataType dataType : DataType.values()) {
+//            dataTypeComboBox.getItems().add(dataType);
+//        }
+//        dataTypeComboBox.setValue(readOnlyStringControl.getType());
+//        dataTypeComboBox.valueProperty().addListener(observable -> {
+//            readOnlyStringControl.setType(dataTypeComboBox.getValue());
+//        });
```

Comment:
Is this for notes. If not, should we remove?

Reply:
The sampler got slightly broke with these changes because read only component is now generic. This if for notes so that I change the sampler later to make it work. Main issue is that you can't define the generic type of a control in fxml but I can do a workaround later to make that work.